subsys/bluetooth/mesh/shell/shell.c

```diff
@@ -199,125 +199,199 @@ struct bt_mesh_health_cli bt_mesh_shell_health_cli = {
 	.attention_status = health_attention_status,
 	.period_status = health_period_status,
 };
-#endif /* CONFIG_BT_MESH_HEALTH_CLI */
+#endif /* CONFIG_BT_MESH_SHELL_HEALTH_CLI */
 
-static int cmd_input_num(const struct shell *sh, size_t argc, char *argv[])
+static int cmd_init(const struct shell *sh, size_t argc, char *argv[])
+{
+
+	ctx_shell = sh;
+	shell_print(sh, "Mesh shell initialized");
+
+	(void)ctx_shell;
```

Comment:
why is it required? `ctx_shell` and `default_key` are global and in use.

Reply:
I think this was needed when mesh shell was enabled without any additional options. Not needed now, I changed linkage of these 2 to make them usable in other places.